### ISSUE TITLE
Failed to get device properties, error code: 3  ??????

### ISSUE THREAD

**Issue by aidenzdly** [keras-team/keras#12794]
My code:

def auc(y_true, y_pred):
    auc = tf.metrics.auc(y_true, y_pred)[1]
    K.get_session().run(tf.local_variables_initializer())
    return auc

global graph
graph = tf.get_default_graph()
model = load_model(r"h5Model/retrain_model.h5", custom_objects={'auc': auc})

@app.route('/predict', methods=['POST'])
def predict():
    loadImgDir = '/home/tf-serving/imgO/'
    loadImgPath = [os.path.join(loadImgDir, i) for i in os.listdir(loadImgDir)]
    for perImgPath in loadImgPath:
        perImgName = perImgPath.split('/')[-1]
        if perImgPath == None:
            return 'false path'
        if not os.path.exists(perImgPath):
            return "filePath not exist!"
        imgRead = Image.open(perImgPath).resize((32, 32), Image.BILINEAR)
        x = np.array(imgRead, dtype='float32')
        x = np.expand_dims(x, axis=0)
        # print("modelmodel",model.summary())
        with graph.as_default():
            results = model.predict(x)
            label_result = label[np.argmax(results)]
            print(f'maybe:{label_result},num:{np.max(results)}', )

        imgRead = cv2.imread(perImgPath)
        fontPath = 'static/font/simsun.ttc'
        font = ImageFont.truetype(fontPath, 15)
        imgPil = Image.fromarray(imgRead)
        draw = ImageDraw.Draw(imgPil)
        draw.text((100, 50), 'maybe：%s,num：%.2f%%' %
                  (label_result, np.max(results) * 100), font=font,
                  fill=(0, 0, 0))
        imgRead = np.array(imgPil)
        cv2.imwrite(('/home/tf-serving/imgT/%s' % perImgName), imgRead)
    return 'success！'

when i post this url,normal model deployment，but when i predict，error information：

E tensorflow/core/grappler/clusters/utils.cc:83] Failed to get device properties, error code: 3

what can i do？Please help me.
